easybuild/easyconfigs/p/PGI/PGI-16.1-GCC-4.9.2-2.25.eb

```diff
@@ -0,0 +1,26 @@
+name = 'PGI'
+version = '16.1'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgicdk-%(version_major)s%(version_minor)s.tar.gz']
```

Comment:
@valtandor I guess a `versionsuffix` like `-cdk` would make sense?

Although, we would need a better view on the actual differences between the `cdk` and `linux` variants first... Is one a superset of the other?

@bartoldeman, @damianam, @ocaisa, @rjeschmi: any ideas?